libs/wire-api/src/Wire/API/Message.hs

```diff
@@ -359,7 +359,7 @@ data ClientMismatchStrategy
 
 protolensToClientMismatchStrategy :: Maybe Proto.Otr.QualifiedNewOtrMessage'ClientMismatchStrategy -> Either String ClientMismatchStrategy
 protolensToClientMismatchStrategy = \case
-  Nothing -> Right MismatchReportAll
+  Nothing -> Left "ClientMismatchStrategy not specified!"
```

Comment:
Is this backwards compatible with old clients who use protobuf on the old endpoint? Or is the unqualified endpoint making use indirectly of this function, too?

Reply:
The unqualified endpoint is not using this at all.